manifests/openssl/params.pp

```diff
@@ -7,8 +7,14 @@
 ) {
   assert_private()
   include '::simp_options'
+
   if $::simp_options::fips or $facts['fips_enabled'] {
-    $cipher_suite = ['FIPS', '!LOW']
+    if $facts['fips_ciphers'] {
+      $cipher_suite = $facts['fips_ciphers']
+    }
+    else {
+      $cipher_suite = ['FIPS', '!LOW']
```

Comment:
If fips_ciphers fact does not exist and this value is used, the system will break.